kickstarter/android-oss
# NTV-191: rewards missing fields

# 📲 What

- the fragment was ready to work with addons, some details where missing for rewards.
# 👀 See
- Rewards carousel
![RewardsCarousel](https://user-images.githubusercontent.com/4083656/133653998-f680f474-4919-4547-a6cf-90f1e8b386a5.png)

- Creator View
![Creator-View](https://user-images.githubusercontent.com/4083656/133654114-5d7e9004-8f57-474a-95ae-505ab9e4abe6.png)

- Project state + friends 
![friendslist](https://user-images.githubusercontent.com/4083656/133656780-e802c39c-b92a-40c0-8cfd-990da26c60f2.png)


|  |  |

# 📋 QA
- Scroll over the rewards carousel
- navigate over several projects
- load the project page as creator
- ... navigate as usual, you should not appreciate any change compared to previous app version

# Story 📖

[NTV-191](https://kickstarter.atlassian.net/browse/NTV-191)
